packages/@aws-cdk/aws-apigateway/test/restapi.test.ts

```diff
@@ -1096,4 +1096,21 @@ describe('restapi', () => {
       DisableExecuteApiEndpoint: true,
     });
   });
+
+  test('"disableExecuteApiEndpoint" can disable the default execute-api endpoint for SpecRestApi', () => {
```

Comment:
Let's also make sure that this properly sets this as false when it's false and that it's set as false when the property isn't set.